content/docs/use-cases/shared-development-server.md

```diff
@@ -3,102 +3,110 @@
+speed for everyone – similar to `git checkout` for your code.
 
 ![](/img/shared-server.png)
 
 ## Preparation
 
-Create a shared directory to be used as the <abbr>cache</abbr> location for
-everyone's <abbr>DVC projects</abbr>, so that all your colleagues can use the
-same project cache:
+Create a directory external to your <abbr>DVC projects</abbr> to be used as a
+shared <abbr>cache</abbr> location for everyone's projects:
 
 ```dvc
-$ mkdir -p /path/to/dvc-cache
+$ mkdir -p /home/shared/dvc-cache
 ```
 
-You will have to make sure that the directory has proper permissions setup, so
-that all your colleagues can read and write to it, and can access cache files
-written by others. The most straightforward way to do this is to make sure that
-everyone's users are members of the same group, and that your shared cache
-directory is owned by this group, with the aforementioned permissions.
+> The `/home/shared` directory used as example above is typical in Linux
+> distributions.
 
-## Transfer existing cache (Optional)
+Make sure that the directory has proper permissions, so that all your colleagues
+can write to it, and can read cached files written by others. The most
+straightforward way to do this is to make all users members of the same group,
+and have the shared cache directory owned by that group.
 
-This step is optional. You can skip it if you are setting up a new DVC project
-whose cache directory is not stored in the default location, `.dvc/cache`. If
-you did work on your project with DVC previously and you wish to transfer your
-cache to the shared cache directory (external to your workspace), you will need
-to simply move it from an old cache location to the new one:
+## Transfer existing cache (optional)
+
+You can skip this part if you are setting up a new DVC project where the local
+<abbr>cache directory</abbr> (`.dvc/cache` by default), hasn't been used.
+
+If you did work on the <abbr>DVC projects</abbr> previously and wish to transfer
+its existing cache to the shared cache directory, you will simply need to move
+its contents from the old location to the new one:
 
 ```dvc
-$ mv .dvc/cache/* /path/to/dvc-cache
+$ mv .dvc/cache/* /home/shared/dvc-cache
 ```
 
-Now you need to ensure that cache files/directories have appropriate
-permissions, so that they could be accessed by your colleagues that are members
-of the same group:
+Now, ensure that the cached directories and files have appropriate permissions,
+so that they can be accessed by your colleagues (assuming their users are
+members of the same group):
 
 ```dvc
-$ sudo find /path/to/dvc-cache -type f -exec chmod 0664 {} \;
-$ sudo find /path/to/dvc-cache -type d -exec chmod 0775 {} \;
-$ sudo chown -R myuser:ourgroup /path/to/dvc-cache/
+$ sudo find /home/shared/dvc-cache -type d -exec chmod 0775 {} \;
+$ sudo find /home/shared/dvc-cache -type f -exec chmod 0664 {} \;
+$ sudo chown -R myuser:ourgroup /home/shared/dvc-cache/
```

Comment:
I'm not sure these commands will always work. What if the shared cache dir has some files your user can't access (if someone didn't do this process before, for example)? Wouldn't the command fail?